src/midi.py

```diff
@@ -1,19 +1,147 @@
-class MidiInstrumentBox(...):
-    def __init__(self, *args, **kwargs) -> None: ...
+from PyQt6.QtWidgets import QLabel, QVBoxLayout, QWidget
 
-    @property
-    def value(self) -> str: ...
+class MidiInstrumentBox(QWidget):
+    def __init__(self):
+        super().__init__()
+        layout = QVBoxLayout(self)
+        self.instrument_label = QLabel()
+        layout.addWidget(self.instrument_label)
 
-    # Texto contido dentro da caixa de texto.
+        self.instrument_names = self._get_instrument_names()
+        self.set_instrument(0)  
 
-    def set_instrument(self, instrument: str) -> None: ...
+    def set_instrument(self, instrument: int):
+        instrument_name = self.instrument_names.get(instrument, "Desconhecido")
+        self.instrument_label.setText(f"Instrumento: {instrument_name}")
 
-    # Nome auto explicativo.
-
-    def display(self) -> None: ...
-
-    # Mostrar o texto para o usuário.
-
-    def hide(self) -> None: ...
-
-    # Esconder o texto do usuário.
+    def _get_instrument_names(self):
+        return {
```

Comment:
Fazendo como a sugestão abaixo, você não precisaria setar a variável `self.instrument_names`.

```suggestion
    @property
    def _instrument_names(self) -> dict[int, str]:
        return {
```